rust/operator-binary/src/main.rs

```diff
@@ -13,10 +13,11 @@ use stackable_operator::CustomResourceExt;
 
 mod built_info {
     include!(concat!(env!("OUT_DIR"), "/built.rs"));
+    pub const TARGET_PLATFORM: Option<&str> = option_env!("TARGET");
 }
 
 #[derive(Parser)]
-#[clap(about = built_info::PKG_DESCRIPTION, author = stackable_operator::cli::AUTHOR)]
+#[clap(about, author)]
```

Comment:
This is already done: https://github.com/stackabletech/commons-operator/blob/c276cec29108770c570f5d519e23458fc4ef482c/rust/operator-binary/src/main.rs#L26